packages/jest-runtime/src/index.ts

```diff
@@ -173,7 +173,7 @@ const supportsNodeColonModulePrefixInImport = (() => {
   return stdout === 'true';
 })();
 
-// consider "node" condition as well - maybe switching on `global.window` in the test env?
+// consider `node` & `browser` condition as well - maybe switching on `global.window` in the test env?
```

Comment:
Maybe there could be enabled by `jest-environment` packages? I don't know if the internal architecture makes it impossible, but from my (external, user) point of view it's what makes more sense.

Reply:
Definitely a possibility! Main worry is `browser` meaning both ESM either CJS, but I guess we can add the option and just not use it ourselves for now

Reply:
I'll open up a separate PR with that after landing this 🙂